modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -406,6 +406,16 @@ public void setUp() throws Exception {
 			_siteScopedObjectDefinition1, _OBJECT_FIELD_NAME_1,
 			_OBJECT_FIELD_VALUE_1);
 
+		_siteScopedObjectDefinition2 =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				Arrays.asList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, true, null,
+						RandomTestUtil.randomString(), _OBJECT_FIELD_NAME_1,
```

Comment:
Maybe better to use a diferent field name from the `_siteScopedObjectDefinition1`, as in the company scoped object definition, for example `_OBJECT_FIELD_NAME_2`

Reply:
in the end we are checking whether the array has a correct number of nested object created, but since we check the field too, it's good to vary it yes :D I will apply the change